teams/tikv/team.json

```diff
@@ -0,0 +1,111 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+
+    "maintainers": [
```

Comment:
@innerr should also be included as maintainer.

Reply:
Definitely, updated.